### PR TITLE
feat: history component and historyType guard with history page added

### PR DESCRIPTION
**Pull request by mehulsatardekar**
files To check 👍 
src/components/history/History.tsx
  src/components/history/historyTypes.ts
  src/pages/history/HistoryPage.tsx
 src/pages/history/historypage.css

email-id = test@gmail.com
password = mehulSATARDEKAR1!

live url 👍 

https://storied-croissant-872b39.netlify.app/history

### CODE REVIEW

File: src/components/history/historyTypes.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { PostgrestError } from "@supabase/supabase-js";
+
+type userHistoryTypes = {
```
Comment (mansipadgelwar) [inline src/components/history/historyTypes.ts:3]: Use PascalCase for type names

**Review by mansipadgelwar — APPROVED** [review]
Use PascalCase for type names. Rest looks good. you can merge your PR.

File: src/components/history/historyTypes.ts (hunk 1 above)
Comment (mehulsatardekar) [reply src/components/history/historyTypes.ts:3]: @mansipadgelwar any reasons for suggesting using Pascal over Camel case for types?

Comment (mansipadgelwar) [reply src/components/history/historyTypes.ts:3]: coding conventions

**Review by Naman-Saxena1 — APPROVED** [review]
Code quality and syntax looks good 👍

File: src/components/history/History.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  return (
+    <section>
+       <div className="flex flex-between  flex-align-item-center pb-1">
+           <h1>Your Progress</h1>
+           <h3 className='font-ex-sm'>Scores</h3>
+       </div>
+       <hr />   
+       {
+        (userQuizHistories.length>0)?(
+            userQuizHistories.map(userQuizHistory =>{
```
Comment (rahulyadav139) [inline src/components/history/History.tsx:41]: Suggestion: you can use object destructuring here.

**Review by rahulyadav139 — APPROVED** [review]

**Pull request merged**